src/plugins/discover/public/application/view_alert/view_alert_route.tsx

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+import React, { useEffect, useMemo } from 'react';
+import { useHistory, useLocation, useParams } from 'react-router-dom';
+import { sha256 } from 'js-sha256';
+import { i18n } from '@kbn/i18n';
+import { ToastsStart } from 'kibana/public';
+import type { Alert } from '../../../../../../x-pack/plugins/alerting/common';
+import { AlertTypeParams } from '../../../../../../x-pack/plugins/alerting/common';
```

Comment:
Think it's also a type, right?
```suggestion
import type { AlertTypeParams } from '../../../../../../x-pack/plugins/alerting/common';
```